### DISCUSSION TITLE
The ROCm software naming convention needs standardized

### DISCUSSION THREAD

**Discussion by YellowRoseCx** [ROCm/ROCm#2742] [Ideas]
The ROCm software naming convention has not been standardized for several releases, this can cause confusion and user misguidance when trying to install the software or simply to refer to the software in the correct term.

For example, the latest ROCm software is named ROCm software 6.0.0 In both the release file on GitHub and in the documentation. However, within the AMD ROCm package directory, it is simply named "6.0"

In the release notes of ROCm 6.0.0, the main document title and first page subtitle are in direct odds with each other:
"Release notes for AMD ROCm™ 6.0" is the title of the page content,
However, "ROCm™ Software 6.0.0" is the name of the release and document header.

This also happens in previous releases of the software.

Looking back at previous releases that end in a ".0", You will see that it changes between including ".0" within the name and it not including ".0" within the name. 

Within the APT directory for ROCm on the AMD website, the last time the package link to the folder of the release contained a ".0" at the end of the version number was in "2.9.0"

Hip SDK for Windows is simply named 5.5, but the corresponding Linux release is ROCm 5.5.0. ROCm 5.4 is listed as 5.4 on the release page on GitHub and the package directory, but within the documentation it's labeled 5.4.0 and on the branch for the code, it's listed as 5.4.0.

ROCm 4.0.0 is labeled as 4.0.0 on the GitHub release page and the documentation, but within the package directory it's labeled 4.0.
ROCm 3.10.0 is listed as such on the release page of GitHub and the documentation, but only as 3.10 on the package folder.

--

You get my point, the naming convention of ROCm needs to be standardized so that it conforms to the same name as the software's documented name within all notes, releases, and links including the package links.

Please consider this suggestion as it can lead to discrepancies as exists now within the _develop_ branch and the _6.0.0_ branch and user confusion as seen in this recent issue: https://github.com/ROCm/ROCm/issues/2739

**Comment by YellowRoseCx** [comment]
Bump

**Comment by schung-amd** [comment]
Hi, sorry for the lack of response here. I suspect it would cause some headaches for us to change this internally, but I'll check around to see what barriers there might be to doing this. I do sympathize with this request, and this is something I've personally run into while writing some automation.

**Reply by YellowRoseCx** [reply]
Thank you!

**Comment by saadrahim** [comment]
@Danny213123 can you look into this?
